interfaces/cython/cantera/test/test_transport.py

```diff
@@ -88,6 +88,17 @@ def test_add_species_multi(self):
         self.assertNear(gas1.thermal_conductivity, gas2.thermal_conductivity)
         self.assertArrayNear(gas1.multi_diff_coeffs, gas2.multi_diff_coeffs)
 
+    def test_species_visosities(self):
+
+        state = 500, 2e5, 'H2:0.3, O2:0.28, CH4:0.02, H2O:0.3, N2:0.1'
+        gas1 = ct.Solution(thermo='IdealGas', species=[
+            s for s in ct.Species.listFromFile('gri30.xml')])
+        gas1.transport_model = 'Multi'
+        gas1.TPX = state
+
+        self.assertTrue(all(gas1.species_viscosities > 0))
+        self.assertTrue(all(gas1['H2'].species_viscosities > 0))
```

Comment:
A better test here might be to check that the values in `gas1.specie_viscosities` are the same thing you get from `gas.viscosity` if you set the gas to a single-species state.